<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stockdlg</class>
 <widget class="QDialog" name="Stockdlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品编号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>商品名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tradecodelineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="qtyspinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="OkBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="qtylabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>添加库存：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fullnamelineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
